types/scheduler/index.d.ts

```diff
@@ -23,7 +23,7 @@ export function unstable_runWithPriority<T>(priorityLevel: number, eventHandler:
 export function unstable_scheduleCallback(priorityLevel: number, callback: FrameCallbackType, options?: { delay?: number, timeout?: number}): CallbackNode;
 export function unstable_next<T>(eventHandler: () => T): T;
 export function unstable_cancelCallback(callbackNode: CallbackNode): void;
-export function unstable_wrapCallback(callback: FrameCallbackType): () => FrameCallbackType;
+export function unstable_wrapCallback<I extends Array<any>, O>(callback: (...args: I) => O): (...args: I) => O;
```

Comment:
So the it basically just returns the input (which has to be a function). Can't we use a single type parameter in this case instead?

```suggestion
export function unstable_wrapCallback<Callback extends (...args: any[]) => any>(callback: Callback): Callback;
```